src/views/GovernanceReviewTeam/Actions/index.tsx

```diff
@@ -48,7 +50,7 @@ const ActionRadioOption = ({
     <Radio
       {...field}
       className="grt-action-radio__option margin-bottom-2 tablet:grid-col-5"
-      id={`grt-action__${field.value}`}
+      id={`grt-action__${id || field.value}`}
```

Comment:
Would it be clearer overall to use data-testid here? Or in this general situation?

I can see you're doing this to target from cy tests, but this line on its own is a bit strange though not a big deal.

Reply:
The main reason for this was that the ID was weird on a couple of the options. The id originally defaulted to `field.value`, which is the action form route. So some of the options had IDs like `grt-action__actions/action-name`. I could have still targeted the options by ID in the cypress tests without this change.

It probably would have been cleaner to make `id` a required prop instead of having it set in the component with `id || field.value`.